src/components/learner-credit-management/data/hooks/useBudgetContentAssignments.js

```diff
@@ -68,31 +68,31 @@ const useBudgetContentAssignments = ({
   const currentArgsRef = useRef(null);
   const [isLoading, setIsLoading] = useState(true);
   const [contentAssignments, setContentAssignments] = useState(initialContentAssignmentsState);
-  const fetchContentAssignments = useCallback((args) => {
+  const fetchContentAssignments = useCallback(async (args) => {
     if (!isEnabled || !assignmentConfigurationUUID) {
       setIsLoading(false);
       return;
     }
-    const getContentAssignments = async () => {
+
+    try {
       setIsLoading(true);
       const options = {
-        page: args.pageIndex + 1, // `DataTable` uses zeo-indexed array
+        page: args.pageIndex + 1, // `DataTable` uses zero-indexed array
         pageSize: args.pageSize,
       };
       applyFiltersToOptions(args.filters, options);
       applySortByToOptions(args.sortBy, options);
 
-      /* This logic in conjunction with useRef is being used to prevent track events
-      from being called when the page re-renders without the specifically selected
-      arguments (argCopy) being changed */
```

Comment:
[nit/curious] What is the intent for removing this comment? Is it no longer relevant? If it is still relevant, seems like useful context to keep in place?